src/config/network-filesystems.md

```diff
@@ -0,0 +1,70 @@
+# Network Filesystems
+
+## Mounting an NFS Share
+
+To mount an NFS share, start by installing the `nfs-utils` and `netmount`
+packages.
+
+Before mounting an NFS share, [enable](./services/index.md#enabling-services)
+the `statd`, `procbind`, and `netmount` services by linking them into
+`/var/service/`.
```

Comment:
You can stop at `enable the ... services.`

The linking part is explained in the service page.